vms/authentication/urls.py

```diff
@@ -1,14 +1,14 @@
 # Django
-from django.conf.urls import patterns, url
+from django.urls import path
 from django.contrib.auth import views as auth_views
 
 # local Django
 from authentication import views
 from authentication.views import anonymous_required
 
-
-urlpatterns = patterns('',
-    url(r'^$', views.index, name='index'),
-    url(r'^login/$', anonymous_required(auth_views.login), {'template_name': 'authentication/login.html'}, name='login_process'),
-    url(r'^logout/$', auth_views.logout, {'template_name': 'home/home.html'}, name='logout_process'),
-)
+app_name='authentication'
+urlpatterns = [
+    path('', views.index, name='index'),
+    path('login/', anonymous_required(auth_views.login), {'template_name': 'authentication/login.html'}, name='login_process'),
```

Comment:
This line is too long actually. later when we integrate pep8 checks, you will have to change it.

Reply:
I have made the changes you suggested. Please review now.

Reply:
Looks good now. I think the next big thing would be to get this pass the `CI`